chainer/initializers/bilinear.py

```diff
@@ -0,0 +1,53 @@
+import numpy
+
+from chainer.backends import cuda
+from chainer import initializer
+
+# Original code from Berkeley FCN
+# https://github.com/shelhamer/fcn.berkeleyvision.org/blob/master/surgery.py
+
+
+def _get_upsampling_filter(size):
+    """Make a 2D bilinear kernel suitable for upsampling"""
+    factor = (size + 1) // 2
+    if size % 2 == 1:
+        center = factor - 1.
+    else:
+        center = factor - 0.5
+    og = numpy.ogrid[:size, :size]
+    filter = (1. - abs(og[0] - center) / factor) * \
+             (1. - abs(og[1] - center) / factor)
+    return filter
+
+
+class Bilinear(initializer.Initializer):
```

Comment:
Do you have a reference to call this initializer "Bilinear"?  Don't we need to include `Upsampling` to its name?

Reply:
I just follow caffe's name. it is called `BilinearFiller` in caffe.
http://caffe.berkeleyvision.org/doxygen/classcaffe_1_1BilinearFiller.html

Reply:
I know there is a layer called `Bilinear` in chainer.
 if you think it is confusing, i can change the name.

Reply:
I guess most people use the Bilinear link in the way like L.Bilinear(), so it would be OK to use Bilinear for the name of this initializer.